.travis.yml

```diff
@@ -12,7 +12,7 @@ script:
   - yarn test
 after_success:
   - yarn run coveralls
-  - 'stable'
+  - "stable"
 notifications:
   email: false
   slack: andela:LX8VPRMaSP4tskr94puPVmFf
```

Comment:
Is this the token Alpha taked about in our slack channel??

Reply:
maybe!!!